docs/source/_static/hide_chapter_titles.css

```diff
@@ -1,3 +1,25 @@
+/* front page: hide chapter titles
+ * needed for consistent HTML-PDF-EPUB chapters
+ */
 #how-to-read-this-document.section div.section {
     display:none;
 }
+
+/* parameter files: only show short intro of title
+ * and description in HTML.
+ * Work-Around for https://github.com/michaeljones/breathe/issues/318
```

Comment:
QUESTION:
Is this a workaround by @michaeljones then use `by` instead of `for`.

Reply:
read the issue - `for` is perfectly fine